pysatSpaceWeather/tests/test_instruments.py

```diff
@@ -63,3 +66,44 @@ def setup_method(self):
 
     def teardown_method(self):
         """Runs after every method to clean up previous testing."""
+
+
+class TestDeprecation():
+    def setup(self):
+        """Runs before every method to create a clean testing setup"""
+        warnings.simplefilter("always", DeprecationWarning)
+        self.in_kwargs = {"platform": 'sw', "name": 'f107', "tag": 'all',
```

Comment:
```suggestion

        # Register modules to be tested
        pysat.utils.registry.register_by_module(pysatSpaceWeather.instruments)
        
        self.in_kwargs = {"platform": 'sw', "name": 'f107', "tag": 'all',
```

Reply:
I thought it would be simpler to just use the `inst_module` kwarg instead.  Thanks for the catch!